src/client/src/ui/blotter/model/blotterModel.js

```diff
@@ -60,12 +60,16 @@ export default class BlotterModel extends ModelBase {
           this.router,
           this.modelId,
           (trades:Array<Trade>) => {
+            _.forEach(this.trades, (trade:Trade) => {
```

Comment:
You might want to move this into a method calle `preProcess()` ([doco])(http://esp.readthedocs.org/en/latest/advanced-concepts/complete-event-workflow.html#pre-event-processing). 
Pre processing happens before any change to the model, it allows for any transient state to be reset. 

This way any change to the model will below away the transient status `isNew`. We may not have anything else changing the blotter model ATM, however this could change if we had filtering or such things built in.